domains/cards/services.ts

```diff
@@ -0,0 +1,30 @@
+import { nanoid } from "nanoid";
+import { Card } from "./types";
+
+
+ function createCard(name: string, dateOfBirth: string, gender: string) {
+    let newCard: Card
+  
+    return (newCard = {
+      id: nanoid(),
+      name: name,
+      dateOfBirth: dateOfBirth,
+      gender: gender,
+    });
+  }
+
+  function deleteCard (id:string) {
+    let allCardsData = JSON.parse(localStorage.getItem("cards"));
+    const deleteCardWithId = allCardsData.find((e: Card)=>e.id ===id)
+    let index = allCardsData.indexOf(deleteCardWithId)
+    allCardsData.splice(index, 1) 
+    localStorage.setItem("cards", JSON.stringify(allCardsData));
+}
+
+function showCard (id: string){
+    let allCardsData = JSON.parse(localStorage.getItem("cards"));
+    const CardWithId = allCardsData.find((e: Card)=>e.id ===id);
+     return CardWithId
```

Comment:
Краще використовувати `camelCase`: `CardWithId => cardWithId`.

А в цьому випадку можна одразу повернути значення.

```suggestion
    return allCardsData.find((e: Card) => e.id === id);
```